pyproject.toml

```diff
@@ -96,7 +95,6 @@ exclude = ["*.tests", "*.tests.*", "tests.*", "tests"]
 namespaces = false
 
 [tool.black]
-target-version = ['py310']
```

Comment:
This is cruft from before we had python support made explicit via `requires-python`